templates/00-hello-world/src/local_test_environment.js

```diff
@@ -0,0 +1,82 @@
+const NodeEnvironment = require('jest-environment-node');
+const dev = require('nearlib/dev');
+const fs = require('fs');
+const nearlib = require('nearlib');
+
+
+class LocalTestEnvironment extends NodeEnvironment {
+    constructor(config) {
+        super(config);
+    }
+
+    async setup() {
+        this.global.ArrayBuffer = ArrayBuffer;
```

Comment:
hmm, why is it needed to copy all the already global stuff to `this.global`?

Reply:
There is some obscure buggy behaviour of node test environment and versions of dependencies it uses. Something in buffers starts breaking. Sadly this is the recommended solution.

Reply:
>  Sadly this is the recommended solution.

May you insert link to this? Would help whoever is reading code and has same question.